seed_db.sh

```diff
@@ -0,0 +1,136 @@
+# ***** How to Use: *****
+# For now, you will need to set the two secret variables $USER_AUTH_ID and $ADMIN_AUTH_ID in your own shell.
+# The values of these variables can be found in the secret channel. 
+# Working to add these to vault and, in a future update, you won't need to set these variables manually. 
+# 
+# In the directory of the script, call ./seed_db.sh [OPTION] 
```

Comment:
nit: Just to make things crystal clear, can you add an example like

`USER_AUTH_ID=123 ADMIN_AUTH_ID=1234 ./seed_db.sh kevin`